composables/use-database.ts

```diff
@@ -21,16 +22,34 @@ export interface ReviewType {
   startDate: string;
   slug: string;
   schema: SchemaType[];
-  entry: EntryType;
 }
 
 interface SchemaType {
+  widgetId: string;
+}
+
+interface WidgetsType {
+  widgetTypes: WidgetType[];
+  existingWidgets: ExistingWidgetType[];
+}
+
+interface WidgetType {
+  id: string;
   title: string;
+  type: "number" | "string";
+}
+
+export interface ExistingWidgetType {
```

Comment:
#### TL;DR:
I suggest we remove `data` from `ExistingWidgetType` and define a third type, let's call it `ExistingWidgetTypeData`, that exists on each review using this widget.

#### Full explanation
I believe we need 3 types altogether to cover our use-cases:
- One for all different types of widgets (Manual number, Manual string, MTTR, JIRA Issue count etc.) - this is exactly like `WidgetType` in this PR
- One for a configured instance of a specific widget type, something like:
   ```js
   {
     type: "mttr", // referencing a WidgetType object
     id: "team-broccoli-mttr",
     label: "Team Broccoli MTTR", 
     pql: "team=brocolli AND repo=sleuth" 
   }
   ```
   This is almost like `ExistingWidgetType` in this PR, but without `data`.
- One to tie a specific report and an `ExistingWidgetType` together + carry the entered data. This probably exists right on the review.
   ```js
   review: {
     data: [
       widget: "team-broccoli-mttr",
       values: [{ 
         _date: "2024-01-22",
         _user: 1,
         value: 2,
       }, { 
         _date: "2024-01-23",
         _user: 2,
         value: 6,
       }]
     ]
   }
   ```